blackduck/Client.py

```diff
@@ -56,24 +56,33 @@ def __init__(self, base_url, timeout, retries, verify):
 
     def request(self, method, url, **kwargs):
         kwargs['timeout'] = self._timeout
+        if method.lower() == 'get':
+            # set default media type if not provided
+            headers = {
+                'accept': "application/json",
+                'content-type': "application/json"
+            }
+            headers.update(kwargs.pop('headers', dict()))
+            kwargs['headers'] = headers
         url = urljoin(self.base_url, url)
         return super().request(method, url, **kwargs)
 
 
 class Client:
-    '''
-    classdocs
-    '''
-    from .constants import VERSION_DISTRIBUTION, VERSION_PHASES, PROJECT_VERSION_SETTINGS
```

Comment:
These constants are specific to REST API endpoint internals and belong to a higher/different layer. I prefer to keep Client very focused and generic.